k8s.jsonnet

```
@@ -232,8 +241,15 @@
             },
           },
         ],
+        [if ($.config.ingress_tls) then 'tls']: [
+            {
+                hosts: [
+                    $.config.ingress_host,
+                ],
+                [if (std.objectHas($.config, "ingress_secret_name")) then 'secretName']: $.config.ingress_secret_name,
```

`std.objectHas(x, y)` can be written more cleanly as `x in y`
```suggestion
                [if "ingress_secret_name" in $.config then 'secretName']: $.config.ingress_secret_name,
```